COMETwebapp/ViewModels/Components/SystemRepresentation/SystemRepresentationBodyViewModel.cs

```diff
@@ -47,7 +47,7 @@ public class SystemRepresentationBodyViewModel : SingleIterationApplicationBaseV
         /// <summary>
         /// Initializes a new instance of the <see cref="SingleIterationApplicationBaseViewModel" /> class.
         /// </summary>
-        /// <param name="sessionService">The <see cref="ISessionService" /></param>
```

Comment:
Inside the whole file : Indentation problem and revert all changes for <param parameterTypeName, should be <param name="....">